clinvar/src/it/test-files/outputs/clinical_assertion_observation/part-00004.json

```diff
@@ -1,4 +1,4 @@
-{"content":"{\"Sample\":{\"Origin\":{\"$\":\"germline\"},\"Ethnicity\":{\"$\":\"Japanese\"},\"GeographicOrigin\":{\"$\":\"Japan\"},\"Species\":{\"@TaxonomyId\":\"9606\",\"$\":\"human\"},\"Age\":[{\"@Type\":\"minimum\",\"@age_unit\":\"years\",\"$\":\"0\"},{\"@Type\":\"maximum\",\"@age_unit\":\"years\",\"$\":\"9\"}],\"AffectedStatus\":{\"$\":\"yes\"},\"Gender\":{\"$\":\"female\"}},\"Method\":{\"MethodType\":{\"$\":\"research\"}},\"ObservedData\":{\"Attribute\":{\"@Type\":\"VariantAlleles\",\"@integerValue\":\"1\"}}}","id":"SCV000328419.1","clinical_assertion_trait_ids":[]}
-{"content":"{\"Sample\":{\"Origin\":{\"$\":\"maternal\"},\"Species\":{\"@TaxonomyId\":\"9606\",\"$\":\"human\"},\"AffectedStatus\":{\"$\":\"yes\"}},\"Method\":{\"TypePlatform\":{\"$\":\"Oligo aCGH\"},\"Purpose\":{\"$\":\"Discovery\"},\"Description\":{\"$\":\"Microarray\"},\"SourceType\":{\"$\":\"submitter-generated\"},\"MethodType\":{\"$\":\"clinical testing\"}},\"ObservedData\":{\"Attribute\":{\"@Type\":\"VariantAlleles\",\"@integerValue\":\"1\"}},\"XRef\":{\"@DB\":\"dbVar\",\"@ID\":\"nssv580796\",\"@Type\":\"dbVarVariantCallId\"}}","id":"SCV000190832.3","clinical_assertion_trait_ids":["SCV000190832.2.0"]}
-{"content":"{\"Sample\":{\"Origin\":{\"$\":\"not provided\"},\"Species\":{\"@TaxonomyId\":\"9606\",\"$\":\"human\"},\"AffectedStatus\":{\"$\":\"yes\"}},\"Method\":{\"TypePlatform\":{\"$\":\"Oligo aCGH\"},\"Purpose\":{\"$\":\"Discovery\"},\"Description\":{\"$\":\"Microarray\"},\"SourceType\":{\"$\":\"submitter-generated\"},\"MethodType\":{\"$\":\"clinical testing\"}},\"ObservedData\":{\"Attribute\":{\"@Type\":\"VariantAlleles\",\"@integerValue\":\"1\"}},\"XRef\":{\"@DB\":\"dbVar\",\"@ID\":\"nssv580795\",\"@Type\":\"dbVarVariantCallId\"}}","id":"SCV000190832.1","clinical_assertion_trait_ids":["SCV000190832.0.0"]}
-{"content":"{\"Sample\":{\"Origin\":{\"$\":\"not provided\"},\"Species\":{\"@TaxonomyId\":\"9606\",\"$\":\"human\"},\"AffectedStatus\":{\"$\":\"yes\"}},\"Method\":{\"TypePlatform\":{\"$\":\"Oligo aCGH\"},\"Purpose\":{\"$\":\"Discovery\"},\"Description\":{\"$\":\"Microarray\"},\"SourceType\":{\"$\":\"submitter-generated\"},\"MethodType\":{\"$\":\"clinical testing\"}},\"ObservedData\":{\"Attribute\":{\"@Type\":\"VariantAlleles\",\"@integerValue\":\"1\"}},\"XRef\":{\"@DB\":\"dbVar\",\"@ID\":\"nssv580797\",\"@Type\":\"dbVarVariantCallId\"}}","id":"SCV000190831.1","clinical_assertion_trait_ids":["SCV000190831.0.0"]}
+{"content":"{\"Sample\":{\"Origin\":{\"$\":\"germline\"},\"Ethnicity\":{\"$\":\"Japanese\"},\"GeographicOrigin\":{\"$\":\"Japan\"},\"Species\":{\"@TaxonomyId\":\"9606\",\"$\":\"human\"},\"Age\":[{\"@Type\":\"minimum\",\"@age_unit\":\"years\",\"$\":\"0\"},{\"@Type\":\"maximum\",\"@age_unit\":\"years\",\"$\":\"9\"}],\"AffectedStatus\":{\"$\":\"yes\"},\"Gender\":{\"$\":\"female\"}},\"Method\":{\"MethodType\":{\"$\":\"research\"}},\"ObservedData\":{\"Attribute\":{\"@Type\":\"VariantAlleles\",\"@integerValue\":\"1\"}}}","id":"SCV000328419.1"}
```

Comment:
@danxmoran do we want the field to be absent if there is no data to put in it, or do we still want the field present but with some empty value?